nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -1378,6 +1380,13 @@ public String prompt(String message, Predicate<String> predicate, String error)
         return pwd;
     }
 
+    /**
+     * @since 11.1
+     */
+    public char[] promptToken() throws IOException {
+        return promptPassword("Please enter your token: ");
```

Comment:
This creates a deprecation warning.

Reply:
Why? `#promptPassword(String)` is not deprecated 🤔